src/Adapter/Cart/CommandHandler/AddCustomizationFieldsHandler.php

```diff
@@ -110,4 +116,73 @@ public function handle(AddCustomizationFieldsCommand $command): int
 
         return $customizationId;
     }
+
+    /**
+     * Uploads image from customized field
+     *
+     * @param UploadedFile $file
+     *
+     * @return string
+     *
+     * @throws FileUploadException
+     */
+    private function uploadImage(UploadedFile $file): string
+    {
+        $this->validateUpload($file);
+
+        //@todo: check if copy is okay to use instead of move_uploaded_file(this fails creating new request from global later)
+        if (!($tmpName = tempnam(_PS_TMP_IMG_DIR_, 'PS')) || !copy($file->getPathname(), $tmpName)) {
+            throw new FileUploadException('An error occurred during the image upload process.');
+        }
+        $fileName = md5(uniqid(mt_rand(0, mt_getrandmax()), true));
```

Comment:
I think `uniqid(mt_rand(...))` is useless 🤔- we cant remove the mt_rand, we get no more entropy from this